polyfill/test/duration.mjs

```diff
@@ -734,6 +737,22 @@ describe('Duration', () => {
       const almostMicrosecond = Duration.from({ nanoseconds: 999 });
       equal(`${almostMicrosecond.round({ smallestUnit: 'microseconds' })}`, 'PT0.000001S');
     });
+    const hours25 = new Duration(0, 0, 0, 0, 25);
+    it('days are 24 hours if relativeTo not given', () => {
+      equal(`${hours25.round({ largestUnit: 'days' })}`, 'P1DT1H');
+    });
+    it('days are 24 hours if relativeTo is PlainDateTime', () => {
+      const relativeTo = Temporal.PlainDateTime.from('2017-01-01');
+      equal(`${hours25.round({ largestUnit: 'days', relativeTo })}`, 'P1DT1H');
+    });
+    it('days are 24 hours if relativeTo is ZonedDateTime, and duration encompasses no DST change', () => {
```

Comment:
Why is there no test here where a DST change does happen?

Reply:
I realized this is confusing, but I didn't add one because it doesn't work yet for round(). I have written one in my in-progress branch for making this work for round(), total(), add(), and subtract().

Reply:
OK sounds good. Thanks for clarifying.